folding-schemes/src/folding/nova/cyclefold.rs

```diff
@@ -189,13 +182,36 @@ pub struct NIFSFullGadget<C: CurveGroup, GC: CurveVar<C, CF2<C>>> {
     _c: PhantomData<C>,
     _gc: PhantomData<GC>,
 }
+
 impl<C: CurveGroup, GC: CurveVar<C, CF2<C>>> NIFSFullGadget<C, GC>
 where
     C: CurveGroup,
     GC: CurveVar<C, CF2<C>>,
     <C as ark_ec::CurveGroup>::BaseField: ark_ff::PrimeField,
     for<'a> &'a GC: GroupOpsBounds<'a, C, GC>,
 {
+    pub fn fold_committed_instance(
+        // assumes that r_bits is equal to r_nonnat just that in a different format
+        r_bits: Vec<Boolean<CF2<C>>>,
+        r_nonnat: NonNativeFieldVar<C::ScalarField, CF2<C>>,
+        cmT: GC,
+        // ci1 is assumed to be always with cmE=0, u=1 (checks done previous to this method)
+        ci1: CycleFoldCommittedInstanceVar<C, GC>,
+        ci2: CycleFoldCommittedInstanceVar<C, GC>,
```

Comment:
This was a mistake made by me in a previous commit, where this comment should be for `ci2`, which is the one which is assumed to be with `cmE=0` and `u=1`, right? Which comes from `u_i`.
The code of the function already uses it correctly since does
`cmE: cmT.scalar_mul_le(r_bits.iter())? + ci1.cmE`
where `ci2.cmE` is skipped since is `0`. So it would just be updating the comment to `ci2` instead of `ci1`.
(same for the `verify` method, for the comment at line 220)